README.md

```diff
@@ -15,88 +15,31 @@ by its developers, nor is it "supported" software.
 
 ## Table of Contents
 
-- [Community Meetings](#community-meetings)
 - [Quick Start](#quick-start)
+- [Community Meetings](#community-meetings)
+- [Pull Request Review Series](#pull-request-review-series)
 - [Documents](#documents)
 - [Code of Conduct](#code-of-conduct)
 - [Release Management](#release-management)
 - [Licensing](#licensing)
 - [Trademark](#trademark)
 
-## Community meetings
-
-- Agenda: <https://hackmd.io/ABueHjizRz2iFQpWnQrnNA>
-- We hold a weekly Ratify community meeting on Weds 4:30-5:30pm (Pacific Time)   
-Get Ratify Community Meeting Calendar [here](https://calendar.google.com/calendar/u/0?cid=OWJjdTF2M3ZiZGhubm1mNmJyMDhzc2swNTRAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
-- We meet regularly to discuss and prioritize issues. The meeting may get cancelled due to holidays, all cancellation will be posted to meeting notes prior to the meeting.
-- Reach out on Slack at [cloud-native.slack.com#ratify](https://cloud-native.slack.com/archives/C03T3PEKVA9). If you're not already a member of cloud-native slack channel, first add [yourself here](https://communityinviter.com/apps/cloud-native/cncf).
-
-## Pull Request Review Series
-- We hold a weekly Ratify Pull Request Review Series on Mondays 5-6 pm PST.  
-- People are able to use this time to walk through any Pull Requests and seek feedback from others in the Community.  If there are no PR to review, the meeting will be cancelled during that week.
-- Reach out on Slack if you want to reserve a session for review or during our weekly community meetings.
-
 ## Quick Start
 
 Try out ratify in Kubernetes through Gatekeeper as the admission controller.
 
 Prerequisites:
 - Kubernetes v1.20 or higher
-- OPA Gatekeeper v3.10 or higher  
-
-### Step 1: Setup Gatekeeper with [external data](https://open-policy-agent.github.io/gatekeeper/website/docs/externaldata)
-
-> NOTE: If you have added Helm repository for Gatekeeper and Ratify, you can update them by executing `helm repo update` before installation.
-
-```bash
-helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
-
-helm install gatekeeper/gatekeeper  \
-    --name-template=gatekeeper \
-    --namespace gatekeeper-system --create-namespace \
-    --set enableExternalData=true \
-    --set validatingWebhookTimeoutSeconds=5 \
-    --set mutatingWebhookTimeoutSeconds=2
-```
-
-> NOTE: `validatingWebhookTimeoutSeconds` and `mutationWebhookTimeoutSeconds` increased from 3 to 5 and 1 to 2 respectively, so all Ratify operations complete in complex scenarios. See [discussion here](https://github.com/deislabs/ratify/issues/269) to remove this requirement. Kubernetes v1.20 or higher is REQUIRED to increase timeout. Timeout is configurable in helm chart under `provider.timeout` section.   
+- OPA Gatekeeper v3.10 or higher
+- [helmfile](https://helmfile.readthedocs.io/en/latest/#installation) v0.14 or higher (See [ratify-quickstart-manual.md](docs/quickstarts/ratify-quickstart-manual.md) for manual install steps)
```

Comment:
Suggestion add to a bit more information on when to use helmfile. Maybe something like .. " use helm file to install all required dependencies, however you if you are interested to run through whats being installed, refer to the manual steps"